chart/values.yaml

```diff
@@ -101,3 +101,6 @@ resources: {}
 tls: ingress
 
 systemDefaultRegistry: ""
+
+# Set to use the packaged system charts(available as of v2.3.0)
```

Comment:
I think the `(available as of v2.3.0)` is unnecessary given that it's not gonna show up in a pre-2.3 chart.

Reply:
updated